stories/tooltip.stories.js

```diff
@@ -0,0 +1,232 @@
+import React from 'react';
+import { Story, Demo, Description, Code, comment } from './storybook/blocks';
+import './tooltip.scss';
+
+const title = 'Tooltip';
+const component = 'Manifold__Tooltip';
+
+export default { title, component };
+
+export const defaultTooltip = () => (
+  <Story>
+    <Demo>
+      <div class="Manifold__Tooltip__Container" aria-describedby="tooltip-1">
+        <button type="button" className="Manifold__Button">
+          Hover me
+        </button>
+        <div role="tooltip" id="tooltip-1" class="Manifold__Tooltip">
+          Tooltip describing the trigger element.
+        </div>
+      </div>
+    </Demo>
+    <Description>
+      <h1>Default</h1>
+      <p>
+        Use tooltips to provide extra context to the user when hovering a trigger. The tooltip
+        contents default to one line of text.
+      </p>
+      <p>
+        Make sure to add the <code>aria-describedby="tooltip-id"</code> attribute to the element
+        triggering the tooltip. To use the tooltip without the trigger and position manually, omit
+        the `Manifold__Tooltip__Container` wrapping element.
+      </p>
+    </Description>
+    <Code
+      tabs={{
+        html: `<!-- ${title} -->
+<div class="Manifold__Tooltip__Container" aria-describedby="tooltip-1">
+  <div role="tooltip" id="tooltip-1" class="Manifold__Tooltip Manifold__Tooltip--Down">
```

Comment:
This does **NOT** get `className` so this is correct ✅ (I have a few mistakes in my HTML from copy/pasting back-and-forth in React)